### PR TITLE
issue 111 changes made

### PR DESCRIPTION
**Pull request by qualmless**
addresses issue #111 - There are lint errors in HeroComponent that need to be fixed

1. src/app/hero/hero.component.spec.ts:10:14
- [ ] WARNING: 10:14 deprecation async is deprecated: use waitForAsync(), (expected removal in v12)
- [ ]  WARNING: 27:41 deprecation async is deprecated: use waitForAsync(), (expected removal in v12)

2. src/app/hero/hero.component.ts:12:3
- [ ] ERROR: 12:3 typedef expected call-signature: 'ngOnInit' to have a typedef

### CODE REVIEW

File: src/app/hero/hero.component.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export class HeroComponent implements OnInit {
 
   constructor() { }
 
-  ngOnInit() {
+  ngOnInit() : void {
```
Comment (alisaduncan) [inline src/app/hero/hero.component.ts:12]: @qualmless would you mind remove the space before the colon? I'm seeing a new lint error from the white space. You can run `npm run lint` and check that the output doesn't contain any lint error from _hero.component.ts_. Thanks!

**Review by alisaduncan — COMMENTED** [review]
One change please as noted. Good job on updating your fork with latest changes from upstream! Everything else looks great!

**Review by qualmless — COMMENTED** [review]
same question here @alisaduncan!

**Review by alisaduncan — COMMENTED** [review]
Looks like you're missing a change on line 27 of the spec file. This is causing tests to fail. Please refer to the contributions guide on how to verify your changes in this repo. You're almost there!

**Review by alisaduncan — APPROVED** [review]
fixes issue #111

**Pull request merged**